RELEASE.md

```diff
@@ -182,6 +182,37 @@ For that you need to run the following commands:
 
     - Publish the release to Conda-forge by doing a PR against the `rc` branch of the [spyder-feedstock repo](https://github.com/conda-forge/spyder-feedstock).
 
+        - Create branch for the update
+
+              git checkout rc
+              git fetch upstream
+              git merge upstream/rc
+              git checkout -b update_X.X.Xrc1
+
+        - Update `rc` branch in the Spyder feedstock with the latest changes in the `main` one:
+
+            - Create a patch between the branches and apply it
+
+                  git checkout main
+                  git fetch upstream
+                  git merge upstream/main
+                  git diff main rc > update.patch
+                  patch -p1 -R < update.patch
+
+            - Revert changes to every file outside the `recipe` directory, except for `conda-forge.yml`. They will be applied when re-rendering the feedstock.
+
+            - Restore `recipe/conda_build_config.yaml` (that file contains the info to use a different channel).
```

Comment:
Not completly sure of the syntax but maybe using something like

```
git diff main rc recipe/ ":!recipe/conda_build_config.yaml" > recipe.patch

``` 

To get the patch for the recipe dir contents and

```
git diff main rc conda-forge.yml > conda-forge.patch
```

to get the patch of the `conda-forge.yml` file?

Maybe it is not possible to mark in a single command the paths to ignore so the patch(s) can be applied without worrying about remove or leaving things @ccordoba12 ?

Reply:
Good idea @dalthviz! I didn't know you could include certain directories and exclude files from the diff.

I'll update the instructions accordingly.

Reply:
@dalthviz, please check again the simplified instructions.